calyx-backend/src/verilog.rs

```diff
@@ -151,7 +373,61 @@ impl Backend for VerilogBackend {
                 "File not found: {}",
                 file.as_path_string()
             ))
-        })
+        })?;
+
+        if check_library_needed(ctx) {
+            let handlers: Vec<Box<dyn LibraryHandlerTrait>> =
+                vec![Box::new(HardFloatHandler)];
+            // Special libraries (like HardFloat) are needed, run Morty to pickle the files
+            let library_bundle =
+                build_library_bundle(ctx, &temp_file, &handlers)?;
+
+            let file_list = derive_file_list(ctx, &temp_file, &handlers)?;
+            let syntax_trees = morty::build_syntax_tree(
+                &file_list, false, // By default, don't strip comments
+                false, // By default, don't ignore unparseable
+                true, // By default, propagate defines from first files to the following files
+                false, // By default, don't force sequential
+            )
+            .unwrap();
+            let top_module = ctx.entrypoint.to_string();
+            let _pickled = morty::do_pickle(
+                None::<&String>, // By default, don't need to prepend a name to all global names
```

Comment:
do you really need the type here? Could you try to see if just `None` works?

Reply:
you are right, it can be removed